packages/riverpod_analyzer_utils/lib/src/analyzer_utils.dart

```diff
@@ -11,11 +11,6 @@ Future<AstNode?> findAstNodeForElement(Element element) async {
   if (inSdk) {
     return null;
   }
-  final libraryName = element.librarySource?.uri.path ?? '';
-  // We don't need to visit AST nodes in flutter to check for dependencies, ref usage, etc
-  if (libraryName.startsWith('flutter/')) {
-    return null;
-  }
```

Comment:
I expect this removal to significantly negatively affect analysis times due to how the ref escape tracking works.

Reply:
I just think that this isn't the correct place to do it. That's not its responsibility

It should be done by the code that invokes this function instead